core/src/main/java/org/bitcoinj/core/TransactionInput.java

```diff
@@ -385,6 +385,14 @@ public boolean isOptInFullRBF() {
         return sequence < NO_SEQUENCE - 1;
     }
 
+    /**
+     * Returns whether this input, if it belongs to a version 2 (or higher) transaction, has
+     * <a href="https://github.com/bitcoin/bips/blob/master/bip-0068.mediawiki">relative lock-time</a> enabled.
+     */
+    public boolean hasRelativeLockTime() {
+        return (sequence & SEQUENCE_LOCKTIME_DISABLE_FLAG) == 0;
```

Comment:
`SEQUENCE_LOCKTIME_DISABLE_FLAG` is not defined, was included in another commit with the same desciption in bitcoinj, 9e7cb1c951f630000b2d322e5bb2af827b597a10

Reply:
fixed.